PencilCase/PencilCase/scripts/Pages/ProductEditor.ts

```diff
@@ -68,6 +68,11 @@ export class ProductEditor extends PageBase {
uqlqOW1dJXh70zMHamqI6edDMgb1PWr2TXgwNODlTz4KiUpkt3T1Nd6fgzqAAOc+fLp4OnOjUKbsu9vcB0I8VdTQzlKc1dK9n4J+DewqavE5s5uCU3Lgvqadm/myABrQVuG4h6W0mc0piVoYvN8/iWQAiYr2dN3c0SyJivZ03dzQGdLHBPXIvpvmiuLHBPXIvpvmgLwAAAAAAAAAAAAAAAGexXtGbu5IhkzFe0Zu7kiGBe4J6nF9R8kWBX4J6nF9R8kWAGexXtGbu5IhkzFe0Zu7kiGBosK7Olb+bJZEwrs6Vv5slgAA2km20ktbb7gB4nTpciDpzY1DDsu+8gVmLQy24KdKOJPXE9a3Wesp5s2OdMccyJxRPa2BPq8WmTE4JCcELVrv8Ad/grdruwAAAAAAAAAO9HUxUtRDGm3De0ST2o0cidLnylNlu8L8dVvgzKnpRRKBwJtQuzaT1O2wC5rMVglroU9o4r6207LMp5s2ZOjcc2NxRPvb2fA8AAAAPqbhiUSbTTumnrTL/D6+GphUuY0pyWtW1NeKM+e5UyOVMUyW7RJ3TA1RExXs6bu5o+0NbBVy9dlMS/VD/K+B8xXs6bu5oDOljgnrkX03zRXFjgnrkX03zQF4AAAAAAAAAAAAAAACixKnnR18yKCTMihdrNQtp6kReq1Pu837HkacAQcHlxy6SJTIIoG427RJp2svEnAAZ7Fe0Zu7kiGTMV7Rm7uSIYGiwrs6Vv5slkChnyqfDJUU6NQrXZPa9b2LvINXikyauhJTlw32p63kBZVeISaXU/1xu/6YWtXz8Cmq6yZVR3jdoU9UK2LNkdtttt3b2tnwAAAAAAAAAAAAAAAAAAAAAAAADrTz46ecpkt2a2rua8GW1VVS6rCZscGpqyihb1p3X9ikPUMyOGCKBRNQxpKJdzs7oDyWOCeuRfTfNFcWOCeuRfTfNAXgAAAAAAAAAAAAAAAAAAAADPYr2jN3ckQy4rcMnVFXHNgjlqGK1k276kl4fA4aGqfPK4vICvbbSTbaSsr9x8LHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVwLHQ1T55XF5DQ1T55XF5AVxY4J65F9N80NDVPnlcXkSsOoJ1LPimTIoGnA1aFtu90+9fACyAAAAAAAAAAAAAAAAAAAAAAAABVYjPqIJkUr0kPRi1pLU0vizlJnT5cxyoaiWrv8AdE7rZ4tAXQKudUVUqR0+sS4240k5dnbU731fI8yp9VNg6TrJMCbtaJpPhYC2BCpW4ZrcytgmOJWUKa2/DX/B7ret3h6pss+ls3bQJQKz/Vf/AHQOEqsrZsxQS5l4nsVkvj4AXQKz/Vf/AHQJNF1v9fWvh0dnxvs3ASgVlfVVEqq9HKmdFWWqyet/NHyomVtPDeOqltvZCkm3usBaAqJVVVTIW3VyoNdrRWT5HpzatQtqskxNJuys20ld21fAC1BXYbVTp86KGbH0kobpWS13+CJNfMjl0kccETUSas18wJAK/C58yb6X00xu1rXezaSayNKkmOGOzS1NPWB3BWYTMiiim+kjbslbpO/iWaabsmm/mAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABWVcubFNimR0stwp2UUUy2ru70V6V51uhL27On+njf+S6xL1CZu5oq6animy3EqdzEna6jStqWqwHiao1KStKhhTvaGNO72X2tnafKp5dFA4Y4Ip110rR3ff3J2OnU4/cn/APZHGpp4pUrpOnctXSu5ie6wE3DqaTFTy5zgvMTbvd7U3bVc8zdJelj9H+zpPo/t2X1bTthnqMHzfM7TqiVISc2NK+xbW9wFfHFicEuKKN2hSu3+jYQpDmQzVHKihhiSbTbS+Hfq7yRPqp1bGpUuG0LeqFPb82fK6lhpZcmFO8TbcTtt2avkB3hixSJJwtNPY04GmdqbSHp4esf09d/2+Dts17bHSinS1Ry25kKsrO7tY5U1fMn1Tl9CFwNtpq6aS73/AGAi4n6+vkjnVKX0o2nOii6VrxqyXw+J0xPVXr5I71cUyZG0lTzJad4U41davmgISimuUlDLlNbLqCFxb+86SnLlSpkMciZ6ZwRJNrUrp93d8zxHIjety5aXhDMX8tkimnwKlnyVA4EoG7xRpttq1krID5g/9eZ/1/lEzFGlRRJva0lxIeD/ANeZ/wBf5R0xafA1DJhd4k7xW7tWz+4ESkkS53S9LOhlWta7Wu9/FkjqNL77BxWZ6pMNgm08Mc5xpxa0k0tXd3HbRVP55vFZAR1QU8TShrIG3qSTTb/uSabD1TzlMUxxNJqzVtpXyYVLxKGBNtQzLK+2yZegAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAeJ8pT5MUuJtJ2u1t1O5EWF06WuKNvxul/BOAELRdP4x8f8Hx4XTtaoo0/G6yJwA508mGRJUuFtpN63tOM+gkT4+m04H3uGyv89RKAHiVJlyYbS4FCu+y1v5vvPNRTy6iX0Jifimtq+R1AEHRVP55vFZHempJVNdy7tva27s7gCLUUMuom+kjjjTslZWPGi6fxj4/4JoAhaLp/GPj/gPC6dp2cafjdZE0ARqaigpo3FBFE21Zp2PjoKdz1M6Frf7UlZv5EoAAABEWHy1U+n6cfS6fStqte9yWAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAf/9k=";
```

Comment:
Tried to use path to load image, but doesn't work. So here use base64 string to load and display image.